packages/text-editor/src/components/extension/table/decorations/optionsButton.ts

```diff
@@ -0,0 +1,30 @@
+import type { Editor } from '@tiptap/core'
+import { type AnySvelteComponent, ModernPopup, showPopup } from '@hcengineering/ui'
+import { optionsSvg } from './icons'
+
+export interface OptionItem {
+  id: string
+  icon: AnySvelteComponent
+  label: string
+  action: () => void
+}
+
+export function createOptionsButton (editor: Editor, options: OptionItem[], className?: string): HTMLElement {
```

Comment:
This won't work for us. Our tables already have massive decorations and cause problems with displaying because we have to keep about 2rem padding on top and left sides of the table.
I think we can use either col/row drag indicator or entire header. But in this case we will need to properly distinguish between click and drag.

Reply:
@aonnikov Moved the options button inside indicators. The button becomes available only when a row or column is selected. The options menu opens on the `click` event, not on `mousedown`, so it doesn't interfere with dragging. Updated the PR description to match